runbotics/runbotics-orchestrator-ui/src/src-landing/views/MarketplaceCartView/MarketplaceCartView.tsx

```diff
@@ -15,11 +15,13 @@ const MarketplaceCartView: FC = () => {
 
     const currentPrice = cart.filter(item => selectedItems.includes(item.slug))
         .reduce(
-            (accumulator, currentValue) => {
-                accumulator = accumulator + ((currentValue.additionalParameters?.basePrice ?? 0) * currentValue.quantity);
-                return accumulator;
-            }, 0,
+            (
+                accumulator,
+                currentValue,
+            ) => accumulator = accumulator + ((currentValue.additionalParameters?.basePrice ?? 0) * currentValue.quantity),
```

Comment:
it could be 
`accumulator += ((currentValue.additionalParameters?.basePrice ?? 0) * currentValue.quantity)`
couldn't it?

i think this makes the order of operations clear